src/configureWorkspace/configurePython.ts

```diff
@@ -66,6 +66,16 @@ gunicorn==20.0.4`;
 const flaskRequirements = `flask==1.1.1
 gunicorn==20.0.4`;
 
+const lowRightsUser = `
+# Switching to a low-rights user, please refer to https://aka.ms/vscode-docker-python-user
```

Comment:
@BigMorty can you look at the text here and on the one below?

Reply:
Docker documentation uses term "non-root" user. Let's use it too for consistency

Reply:
Agreed, "low-rights" (as opposed to "high-rights") is more the Windows terminology.

Reply:
Forgot the reference: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user

Reply:
Changed that to non-root user. Waiting to hear back from @BigMorty and Uche (not sure why I'm unable to tag him) on the wording.

Reply:
Yes I agree, for consistency let's go with "# Switching to a non-root user, please refer to https://aka.ms/vscode-docker-python-user-rights"